bundles/CoreBundle/src/DependencyInjection/Configuration.php

```diff
@@ -926,10 +928,23 @@ private function addImplementationNodeFromArrayDefinition(ArrayNodeDefinition $n
                             ->arrayPrototype()
                             ->children()
                                 ->scalarNode('class')->end()
-                                ->arrayNode('matching')
-                                    ->prototype('scalar')->end()
+                                ->booleanNode('translatable')
+                                    ->defaultTrue()
                                 ->end()
-                            ->end()
+                                ->scalarNode('valid_table')->defaultNull()->end()
+                                ->booleanNode('direct_route')
+                                    ->defaultTrue()
+                                ->end()
+                                ->booleanNode('translatable_inheritance')
+                                    ->defaultTrue()
+                                ->end()
+                                ->booleanNode('children_supported')
+                                    ->defaultTrue()
+                                ->end()
+                                ->booleanNode('only_printable_childrens')
```

Comment:
maybe something more like
```suggestion
                                ->booleanNode('print_container_allowed') //defines if it can be added under a printContainer
```

Reply:
although this should be moved into the web2print bundle, but that might be challenging 🤔

Reply:
This should stay in the Core, that's why we named it `only_printable_childrens`

Reply:
`printcontainers` are not available unless you install and activate web2print bundle, without that, this config would be obsolete, but i recall that we decided to not move yet, just wondering if the change in this PR paves the way for moving it out

Reply:
@mattamon WDYT?

Reply:
In general it should have nothing to do with the web2print bundle, but currently it is only used by the web2print bundle.

Reply:
@kingjia90 we do not have a concept for the subcontex menu yet. So this would be as you guessed very challenging. I think you tried it once. We would have to refactor it like the navigation itself.

ATM this is the best approach as we do not have the dependency on the printcontainer anymore and it can be at least configured.

Reply:
Mhh ok, i was thinking of an arrayNode allowed/disallowed children type or so, since it's now an associative array but let's not change too much

@martineiber Merging as soon as the latest conflict are fixed

Reply:
Fixed the conflict.